homework-3/src/components/CardPatientProfileDoctor/CardPatient.js

```diff
@@ -1,32 +1,44 @@
 import React from 'react';
-import BoxCardPatient from './StylBoxCardPatient';
+import {
+  StylSubTitleCard,
+  StylTitleCard,
+  StylBoxImgPatient,
+  StylBoxColumnInfo,
+  StylBoxRowLine,
+  StylBoxCardPatient,
+} from './index';
+
 import BoxDoctor from '../common/Boxes/StylBoxDoctor';
 import BoxDataPatient from '../Boxes/BoxDataPatient';
 import BoxInfoPatient from '../Boxes/BoxInfoPatients';
-import BoxRow from './StylBoxRowLine';
-import BoxColumnInfo from './StylBoxColumnInfo';
-import BoxImgPatient from './StylBoxImgPatient';
-import TitleCard from './StylTitleCard';
-import SubTitleCard from './StylSubTitleCard';
 import BtnMore from './../BtnsComponents/BtnMore';
+
 import scheduleImg from './../../img/icons/schedule.png';
+import BoxPatientsList from './../../components/common/Boxes/StylBoxPatientsList';
 
-const CardPatient = ({ name, data, info, img, statusText, status, alt }) => {
+const CardPatient = ({ data }) => {
+  console.log(data[0]);
   return (
-    <BoxCardPatient>
-      <BoxRow>
-        <BoxDoctor>
-          <BoxImgPatient src={img} alt={alt} />
-          <BoxColumnInfo>
-            <TitleCard>{name}</TitleCard>
-            <SubTitleCard status={status}>{statusText}</SubTitleCard>
-          </BoxColumnInfo>
-        </BoxDoctor>
-        <BtnMore />
-      </BoxRow>
-      <BoxDataPatient data={data} />
-      <BoxInfoPatient info={info} imgWay={scheduleImg} />
-    </BoxCardPatient>
+    <BoxPatientsList>
+      {data[0].map((item) => (
```

Comment:
data[0] it's your current patient, as i understood data[0] its the object, you should probably pass current Patient into this component, remove map and pass patient.

Reply:
No, data [0] its is an array of objects, so the map method is used